community-build/src/scala/dotty/communitybuild/projects.scala

```diff
@@ -363,6 +363,7 @@ object projects:
     extraSbtArgs  = List("-Dscala.build.compileWithDotty=true"),
     sbtTestCommand   = """set Global / fatalWarnings := false; library/compile""",
     sbtPublishCommand = """set Global / fatalWarnings := false; set library/Compile/packageDoc/publishArtifact := false; library/publishLocal""",
+    scalacOptions = "-Ycompile-scala3-library" :: SbtCommunityProject.scalacOptions
```

Comment:
I am doubting the current usefulness of this `stdLib213` community build test. We have already compile the library in `scala2-library-bootstrapped`. I am considering removing this test at some point. I will propose this formally at a later point in time.